### PR TITLE
Test in more browsers

### PR DESCRIPTION
**Pull request by jrjohnson**
Ensure compatibility with our supported browsers by testing them.

Refs ilios/ilios#4273

### CODE REVIEW

**Comment by jrjohnson** [comment]
Going to live with the failing safari test for now because the fix is going to add a bunch of things to this already big PR. I'll clean it up in a followup ASAP. For now, this is ready to review.

File: tests/integration/components/ics-feed-test.js [hunk 1]
```diff
@@ -17,6 +17,11 @@ module('Integration | Component | ics feed', function (hooks) {
   });
 
   test('copy', async function (assert) {
+    //skip this test if we can't access the clipboard
```
Comment (stopfstedt) [inline tests/integration/components/ics-feed-test.js:20]: nice workaround.

File: tests/integration/modifiers/animate-loading-test.js [hunk 2]
```diff
@@ -8,36 +8,38 @@ module('Integration | Modifier | animate-loading', function (hooks) {
 
   test('it renders defaults', async function (assert) {
     await render(hbs`<div {{animate-loading}}></div>`);
-    assert.dom('div').hasStyle({
-      opacity: '0.1',
-    });
+    assert.ok(
+      Math.abs(getComputedStyle(this.element.querySelector('div'), null).opacity - 0.1) < 0.001
```
Comment (stopfstedt) [inline tests/integration/modifiers/animate-loading-test.js:12]: :exploding_head:

**Review by stopfstedt — APPROVED** [review]

File: tests/integration/modifiers/animate-loading-test.js (hunk 2 above)
Comment (jrjohnson) [reply tests/integration/modifiers/animate-loading-test.js:12]: Yup. I'm not super happy about this, but it's in a test so 🤷

**Pull request merged**